ansible/vtestbed.yaml

```diff
@@ -396,3 +396,19 @@
   inv_name: veos_vtb
   auto_recover: False
   comment: Tests virtual switch vm with a DPU
+
+- conf-name: vms-kvm-smartswitch-ha
+  group-name: vms6-1
+  topo: smartswitch-ha
```

Comment:
Will this work for non-kvm case? If not, it might be better to add KVM in the topo name or something similar.

Reply:
Hi @r12f - this was meant for physical testbeds. I'm working on the internal PR.